dbos/_dbos_config.py

```diff
@@ -61,22 +102,68 @@ class ConfigFile(TypedDict, total=False):
 
     Attributes:
         name (str): Application name
-        language (str): The app language (probably `python`)
         runtimeConfig (RuntimeConfig): Configuration for request serving
         database (DatabaseConfig): Configuration for the application and system databases
+        database_url (str): Database connection string
         telemetry (TelemetryConfig): Configuration for tracing / logging
         env (Dict[str,str]): Environment varialbes
         application (Dict[str, Any]): Application-specific configuration section
 
     """
 
     name: str
-    language: str
     runtimeConfig: RuntimeConfig
     database: DatabaseConfig
+    database_url: Optional[str]
```

Comment:
Should Database be optional as well? We wouldn't expect someone to specify both db and db url, right?

Reply:
I didn't realize `total=False` implied all TypedDict fields are optional. So why are some explicitly marked optional and some aren't?

Reply:
Yeah so the plan for the "FIXME" above is to just make things optional and remove total=False -- not a priority but that'd make the code clearer I think. I don't want to do it in this PR to avoid causing issues I don't know about.